imap/http_caldav.c

```diff
@@ -1672,6 +1672,8 @@ static int export_calendar(struct transaction_t *txn)
     modseq_t syncmodseq = 0;
     int unbind_flag = -1, unchanged_flag = -1;
     struct caldav_db *caldavdb = NULL;
+    unsigned no_declined = !!hash_lookup("noDeclined", &txn->req_qparams);
```

Comment:
We agreed to set C11 as the base line, so we can just as well use type `bool` here.